contributing.md

```diff
@@ -0,0 +1,33 @@
+# Engineering Getting Started
+Welcome! This readme assumes you have already listened to the 311-data pitch, and gone through the basic onboarding. The following will be more geared towards the programming side of 311-data and getting your development environment setup. If you run into any problems, please submit a new issue.
+
+## Feature Branching
+For development we use feature branching to ensure easy collaboration. There aren't any rules to branch naming or how many branches you are allowed to have, but the recommended convention would look like `issueId-Prefix-MinimalDescription`
+For example, a documentation branch could look like `138-DOC-OnboardingUpdate`.
+
+Read more about feature branching [here](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow).
+
+## Default Branch
+Building on feature branching, we treat the 'main' branch as the primary contribution branch. Pull requests to this branch should be as frequent as developers are closing issues (Hopefully very frequent!). Pushes to 'main' will require approval from assigned reviewers before the developer can push and merge their changes.
+
+In other words, whenever you are about to start on a new feature, checkout a new local branch based off of the main branch. Your command would look something like git checkout -b 567-BACK-NewEndpoint. See this stackoverflow post for more context.
```

Comment:
please add the link to the stackoverflow post where it says "See this stackoverflow post for more context"?